FormValidatorSwift.xcodeproj/project.pbxproj

```diff
@@ -25,6 +25,10 @@
 		000A5B6E1E1B9AE5001DC2FF /* ConfigurationSeeds.swift in Sources */ = {isa = PBXBuildFile; fileRef = 000A5B6C1E1B9AE5001DC2FF /* ConfigurationSeeds.swift */; };
 		000D64E91E1F91FD0067B14E /* NumericConfiguration.swift in Sources */ = {isa = PBXBuildFile; fileRef = 000D64E81E1F91FD0067B14E /* NumericConfiguration.swift */; };
 		000D64EA1E1F91FD0067B14E /* NumericConfiguration.swift in Sources */ = {isa = PBXBuildFile; fileRef = 000D64E81E1F91FD0067B14E /* NumericConfiguration.swift */; };
+		00169CDE1E32016400DB3193 /* ValidatorTextFieldTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 88644DA61E2967F7006B0150 /* ValidatorTextFieldTests.swift */; };
```

Comment:
Could we sort the project file? There is a perl script out there (and we keep a shell invoked version internally) which can help or using the Xcode GUI (though there's no easy way to do this for the whole project). It helps reduce potential merge conflicts if multiple people edit the project file by adding new files.